simplq/src/store/index.js

```diff
@@ -1,7 +1,8 @@
 import { combineReducers, configureStore, getDefaultMiddleware } from '@reduxjs/toolkit';
-import queuesReducer from 'store/queues';
+import queuesReducer from 'store/allQueues';
```

Comment:
Stored data (key), slice, and reducer should follow the same naming convention, otherwise itis confusing.

```js
import { combineReducers, configureStore } from '@reduxjs/toolkit';
import allQueuesReducer from 'store/allQueues';

export const rootReducer = combineReducers({
  allQueues: allQueuesReducer,
});

export const store = configureStore({
  reducer: rootReducer,
});
```